althea_types/src/interop.rs

```diff
@@ -119,6 +119,52 @@ pub struct Denom {
     pub decimal: u64,
 }
 
+/// An enum representation of the Regions supported by althea exits
+#[derive(Debug, Clone, Serialize, Deserialize, Eq, PartialEq, Hash)]
+pub enum Regions {
+    US,
+    Canada,
+    PuertoRico,
```

Comment:
PuertoRico can be lumped in with the US for streaming purposes as far as I know. Include Ghana, Nigeria, Columbia (we have or have had networks there)  Mexico etc then drop the list. Don't assign ID's to very large vague continents as we'll just end up having to redefine them to be country specific for the sake of streaming. All of this is about streaming licensing regions

Reply:
done